.nucleus.yaml

```diff
@@ -1,26 +1,61 @@
+# shared config. The `__shared_config__` name isn't meaningful.
+__shared_config__: &branch-definition
+  auto-start: true
+  auto-start-from-forks: false
+  required-downstream-deps:
+    - automation-platform/ui-interaction-explorer-components
+    - communities/microsite-template-marketing
+    - communities/shared-experience-components
+    - communities/ui-b2b-components
+    - communities/ui-cms-components
+    - communities/ui-feeds-components
+    - communities/ui-lightning-community
+    - lwc/lwc-platform
+    - salesforce/lds-lightning-platform
+    - salesforce/luvio
+    - salesforce/lwr
+    - salesforce/lwr-recipes
+    - salesforce/utam-docs
+    - salesforcedevs/developer-website
+    - uiplatform/nucleus
 core-deploy:
   enabled: true
   project-modules:
     lwc: lwc.version
 branches:
   ~DEFAULT~:
     pull-request:
-      auto-start: true
-      auto-start-from-forks: false
-      required-downstream-deps:
-        - automation-platform/ui-interaction-explorer-components
-        - communities/microsite-template-marketing
-        - communities/shared-experience-components
-        - communities/ui-b2b-components
-        - communities/ui-cms-components
-        - communities/ui-feeds-components
-        - communities/ui-lightning-community
-        - lwc/lwc-platform
-        - salesforce/lds-lightning-platform
-        - salesforce/luvio
-        - salesforce/lwr
-        - salesforce/lwr-recipes
-        - salesforce/utam-docs
-        - salesforcedevs/developer-website
-        - uiplatform/nucleus
-
+      <<: *branch-definition
+      merge-method: disabled # do not auto-merge; we'll do it ourselves
```

Comment:
The `merge-method` property can be moved to the `branch-definition` alias. It can then be only overridden on the `release` branch. Here is what I have in [mind](http://nodeca.github.io/js-yaml/#yaml=Y29yZS1kZXBsb3k6CiAgZW5hYmxlZDogdHJ1ZQogIHByb2plY3QtbW9kdWxlczoKICAgIGx3YzogbHdjLnZlcnNpb24KYnJhbmNoZXM6CiAgfkRFRkFVTFR+OgogICAgcHVsbC1yZXF1ZXN0OiAgJmJyYW5jaC1kZWZpbml0aW9uCiAgICAgIGF1dG8tc3RhcnQ6IHRydWUKICAgICAgYXV0by1zdGFydC1mcm9tLWZvcmtzOiBmYWxzZQogICAgICBtZXJnZS1tZXRob2Q6IGRpc2FibGVkICMgZG8gbm90IGF1dG8tbWVyZ2U7IHdlJ2xsIGRvIGl0IG91cnNlbHZlcwogICAgICByZXF1aXJlZC1kb3duc3RyZWFtLWRlcHM6CiAgICAgICAgLSBhdXRvbWF0aW9uLXBsYXRmb3JtL3VpLWludGVyYWN0aW9uLWV4cGxvcmVyLWNvbXBvbmVudHMKICAgICAgICAtIGNvbW11bml0aWVzL21pY3Jvc2l0ZS10ZW1wbGF0ZS1tYXJrZXRpbmcKICAgICAgICAtIGNvbW11bml0aWVzL3NoYXJlZC1leHBlcmllbmNlLWNvbXBvbmVudHMKICAgICAgICAtIGNvbW11bml0aWVzL3VpLWIyYi1jb21wb25lbnRzCiAgICAgICAgLSBjb21tdW5pdGllcy91aS1jbXMtY29tcG9uZW50cwogICAgICAgIC0gY29tbXVuaXRpZXMvdWktZmVlZHMtY29tcG9uZW50cwogICAgICAgIC0gY29tbXVuaXRpZXMvdWktbGlnaHRuaW5nLWNvbW11bml0eQogICAgICAgIC0gbHdjL2x3Yy1wbGF0Zm9ybQogICAgICAgIC0gc2FsZXNmb3JjZS9sZHMtbGlnaHRuaW5nLXBsYXRmb3JtCiAgICAgICAgLSBzYWxlc2ZvcmNlL2x1dmlvCiAgICAgICAgLSBzYWxlc2ZvcmNlL2x3cgogICAgICAgIC0gc2FsZXNmb3JjZS9sd3ItcmVjaXBlcwogICAgICAgIC0gc2FsZXNmb3JjZS91dGFtLWRvY3MKICAgICAgICAtIHNhbGVzZm9yY2VkZXZzL2RldmVsb3Blci13ZWJzaXRlCiAgICAgICAgLSB1aXBsYXRmb3JtL251Y2xldXMKCiAgcmVsZWFzZToKICAgIHB1bGwtcmVxdWVzdDoKICAgICAgPDw6ICpicmFuY2gtZGVmaW5pdGlvbgogICAgICBtZXJnZS1tZXRob2Q6IGZvcmNlLXB1c2ggIyByZWxlYXNlIGJyYW5jaCBzaG91bGQgYWx3YXlzIGJlIGluIHN5bmMgd2l0aCBtYXN0ZXIgYnJhbmNoIChsaW5lYXIgaGlzdG9yeSkKCiAgd2ludGVyMjI6CiAgICBwdWxsLXJlcXVlc3Q6ICpicmFuY2gtZGVmaW5pdGlvbgoKICBzcHJpbmcyMjoKICAgIHB1bGwtcmVxdWVzdDogKmJyYW5jaC1kZWZpbml0aW9uCgpzdGVwczoKICBub2RlLWNvbmZvcm1hbmNlOgogICAgcnVuOgogICAgICBjb21tYW5kOiB5YXJuIHJ1biBsaW50CiAgICBhZnRlcjogbm9kZS1idWlsZAogIG5vZGUtdW5pdC10ZXN0czoKICAgIHJ1bjoKICAgICAgY29tbWFuZDogeWFybiB0ZXN0CiAgIyB0aGlzIHByb2plY3QgcnVucyB5YXJuIGJ1aWxkIGFmdGVyIHlhcm4gaW5zdGFsbCBzbyBza2lwIGV4cGxpY2l0IGJ1aWxkIHN0ZXAKICBub2RlLWJ1aWxkOgogICAgc2tpcDogdHJ1ZQogIG5vZGUtcHJlLXJlbGVhc2UtdGVzdHM6CiAgICBwYXJhbXM6CiAgICAgIGNvbW1hbmQ6IHlhcm4gdGVzdAogIG5wbS1jb25maWd1cmU6CiAgICBwYXJhbXM6CiAgICAgIHJlZ2lzdHJ5LXVybDogaHR0cHM6Ly9yZWdpc3RyeS55YXJucGtnLmNvbQogIG5wbS1jb25maWd1cmUtZm9yLXB1Ymxpc2g6CiAgICBwYXJhbXM6CiAgICAgIHJlZ2lzdHJ5LXVybDogaHR0cHM6Ly9yZWdpc3RyeS5ucG1qcy5vcmc=)

Reply:
This is what I did at first, then I thought it would be neat to pull it out into __shared_config__. But since nucleus doesn't like it, I'll go with your approach.